Application/EdFi.Ods.Common/Validation/ValidationResultExtensions.cs

```diff
@@ -43,7 +44,7 @@ public static string GetAllMessages(this ValidationResult validationResult, int
 
         private static void GetAllMessages(StringBuilder sb, ValidationResult validationResult, string indent)
         {
-            sb.AppendFormat("{0}{1}\n", indent, validationResult.ErrorMessage);
+            sb.AppendFormat($"{{0}}{{1}}{Environment.NewLine}", indent, validationResult.ErrorMessage);
```

Comment:
Also want to document this once and let you take a pass to correct it throughout.  Anytime we're returning out to the end user we should stick with a consistent \n, rather than using Environment.NewLine. This ensures the end user parsing the response doesn't get different results based on different servers.  Environment.NewLine should be for server side uses like logging or nHibernate.